src/azul/openapi/schema.py

```diff
@@ -117,6 +122,29 @@ def array(item: TYPE, *items: TYPE, **kwargs):
     return array_type(make_type(item), *map(make_type, items), **kwargs)
 
 
+def enum(*items: PrimitiveJSON, type_: TYPE = None) -> JSON:
+    """
+    Returns a JSON schema `enum` items of a given type.
+    >>> from azul.doctests import assert_json
+    >>> assert_json(enum('foo', 'bar', type_=str))
+    {
+        "type": "string",
+        "enum": [
+            "foo",
+            "bar"
+        ]
+    }
+    """
+    if type_:
```

Comment:
I will need to address this [requested changes](https://github.com/DataBiosphere/azul/pull/1538/files#r385474794). I am at fault for this.